### PR TITLE
Update dm-stop-task.md

### PR DESCRIPTION
**Pull request by pepezzzz**
add remove task step

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: dm/dm-stop-task.md [hunk 1]
```diff
@@ -68,3 +68,10 @@ stop-task test
     ]
 }
 ```
+
+> **注意：**
+>
+> 同步任务 stop 后，使用 query-status 不能被查询到，但是任务状态仍保存在元数据库中。
```
Comment (qiancai) [inline dm/dm-stop-task.md:74]:
```suggestion
> 使用 `stop-task` 会停止迁移任务的执行，并移除内存中与该任务相关的信息，且不可再通过 [`query-status`](/dm/dm-query-status.md) 进行查询，但不会移除已经写入到下游数据库中的数据以及其中的 checkpoint 等 `dm_meta` 信息。
```

Comment (qiancai) [reply dm/dm-stop-task.md:74]: 以上信息来自 https://docs.pingcap.com/zh/tidb/dev/dm-pause-task
image

File: dm/dm-stop-task.md [hunk 2]
```diff
@@ -68,3 +68,10 @@ stop-task test
...
+>
+> + 对于需要重建的任务，在使用 start-task 时，需要添加 --remove-meta 参数。
+> + 对于需要彻底删除的任务，需要手工删除元数据库 dm_meta 中以表名起始是任务名的四张表。
```
Comment (qiancai) [inline dm/dm-stop-task.md:77]:
```suggestion
> + 对于要重新启动的迁移任务，在使用 [`start-task`](/dm/dm-create-task.md) 时，你需要添加 `--remove-meta` 参数。
> + 对于要彻底删除的迁移任务，你需要在元数据库 `dm_meta` 中删除以该任务名为表名前缀的四张表。
```

**Review by qiancai — APPROVED** [review]

File: dm/dm-stop-task.md [hunk 3]
```diff
@@ -68,3 +68,10 @@ stop-task test
     ]
 }
 ```
+
+> **注意：**
+>
+> 通过 `stop-task` 停止迁移任务后，执行 [`query-status`](/dm/dm-query-status.md) 将无法查询到该任务，但该任务的 checkpoint 等信息仍然保留在元数据库 `dm_meta` 中。
+>
+> + 对于要重新启动的迁移任务，在使用 [`start-task`](/dm/dm-create-task.md) 时，你需要添加 `--remove-meta` 参数。
+> + 对于要彻底删除的迁移任务，你需要在元数据库 `dm_meta` 中删除以该任务名为表名前缀的四张表。
```
Comment (qiancai) [inline dm/dm-stop-task.md:77]:
```suggestion
> + 要重新启动该迁移任务，你需要在 [`start-task`](/dm/dm-create-task.md) 中添加 `--remove-meta` 参数。
> + 要彻底删除该迁移任务，你需要在元数据库 `dm_meta` 中删除以该任务名为表名前缀的四张表。
```

File: dm/dm-stop-task.md [hunk 4]
```diff
@@ -68,3 +68,10 @@ stop-task test
     ]
 }
 ```
+
+> **注意：**
+>
+> 通过 `stop-task` 停止迁移任务后，执行 [`query-status`](/dm/dm-query-status.md) 将无法查询到该任务，但该任务的 checkpoint 等信息仍然保留在元数据库 `dm_meta` 中。
+>
+> + 要重新启动该迁移任务，你需要在 [`start-task`](/dm/dm-create-task.md) 中添加 `--remove-meta` 参数。
```
Comment (qiancai) [inline dm/dm-stop-task.md:76]:
```suggestion
> + 要重新创建该迁移任务，你需要在 [`start-task`](/dm/dm-create-task.md) 中添加 `--remove-meta` 参数。
```

**Comment by pepezzzz** [comment]
LGTM

**Pull request merged**